jicofo/src/main/java/org/jitsi/jicofo/conference/JitsiMeetConferenceImpl.java

```diff
@@ -2325,18 +2325,25 @@ public void bridgeIsShuttingDown(@NotNull Bridge bridge)
         }
 
         @Override
-        public void bridgeRemoved(@NotNull Bridge bridge)
+        public void bridgeFailedHealthCheck(@NotNull Bridge bridge)
```

Comment:
I suggest having a common helper function that both of these call, and passing a reason phrase, would be better than having `bridgeRemoved` call `bridgeFailedHealthCheck` and leaving it ambiguous in the log message.